tutorials/ph-ug-temp/index.md

```diff
@@ -0,0 +1,106 @@
+Using the Earth Engine code editor, users can manipulate these data over a shorter time period. The tool provides LST long-term and short-term time series for user-defined regions and time periods. This tool will restrict data from March 5, 2000 to the present.
+
+1. Pull in a shape file of Uganda. 
+
+In these lines of code, we are creating a new variable called "region". However, now we are pulling in a `FeatureCollection` object, and filtering by ‘Country’ to select ‘Uganda.’ FeatureCollections are groups of features (spatial data and attributes).  `Filter` is the command to extract a specific set of feature data from a feature collection. We then map it.
+  ```
+// Import vector file with country boundaries from Earth Engine
+var dataset = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017'); // All countries
+var uganda_border = dataset.filter(ee.Filter.eq("country_na","Uganda")); 
+// apply filter where "country names" equals "Uganda"
+print(uganda_border) 
+// print to ensure that new "uganda_border" object just contains polygon for Uganda
+
+//Add region outline to layer - for selected countries
+Map.addLayer(uganda_border)
+  ```
+2. Read in land surface data.
+
+Now, we need to read in land surface temperature data. `ImageCollection` objects store collections of images. Using the Earth Engine Data Catalog, we can find different image collections. The `ImageCollection` we are reading in here contains the LST data, but we can read in a different `ImageCollection` for other types of data (e.g. vegetation index, or rain fall, light at night).
+Our code applies filters to restrict images for Uganda, within January to December 2015.
+```
+var modis = ee.ImageCollection('MODIS/MOD11A2'); // read in image collection
+var start = ee.Date('2015-01-01'); // set start date
+var dateRange = ee.DateRange(start, start.advance(1, 'year'));
+// set end date 1 year
+var mod11a2 = modis.filterDate(dateRange).filterBounds(uganda_border);
+// apply filter on date range, uganda boundary
+
+var modLSTday = mod11a2.select('LST_Day_1km'); // pull just 1km day LST
+```
+3. Create a function to convert temperature units.
+
+We then need to convert temperature from Kelvin to Celsius. Note that for remote sensing data, sometimes equations required to convert between different units require additional scaling factors, so we should always make sure to check the [documentation](https://icess.eri.ucsb.edu/modis/LstUsrGuide/usrguide_mod11.html#sds) for the specific product we are using when doing conversions.   After referring to the documentation, we learn that in addition to subtracting 273.15, we must multiply the Kelvin value in our MODIS data by a factor of 0.02.
```

Comment:
Please be more specific to which documentation you're referring to in the link

e.g. "MODIS LST User Guide"

Reply:
Done